doc/how_to_add_a_new_source_file_to_specfem_2D.txt

```diff
@@ -0,0 +1,50 @@
+STEPS TO ADD A SOURCE FILE TO SPECFEM2D
+=======================================
+
+1. Create your source file, for example my_sexy_subroutine.f90 :
+
+      ! Comment here to explain why your subroutine is sexy
+      subroutine my_sexy_subroutine(nothing_funnier_than_that_int)
+
+        use specfem_par, only: nspec,nglob,ibool,coord
+
+        implicit none
+        include "constants.h"
```

Comment:
Fixed!


On 03/13/2015 09:01 PM, Elliott Sales de Andrade wrote:
> In doc/how_to_add_a_new_source_file_to_specfem_2D.txt
> <https://github.com/geodynamics/specfem2d/pull/246#discussion_r26418417>:
>
>> @@ -0,0 +1,50 @@
>> +STEPS TO ADD A SOURCE FILE TO SPECFEM2D
>> +=======================================
>> +
>> +1. Create your source file, for example my_sexy_subroutine.f90 :
>> +
>> +      ! Comment here to explain why your subroutine is sexy
>> +      subroutine my_sexy_subroutine(nothing_funnier_than_that_int)
>> +
>> +        use specfem_par, only: nspec,nglob,ibool,coord
>> +
>> +        implicit none
>> +        include "constants.h"
>
> |specfem_par| includes all constants. There should be no reason to
> include the header as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/246/files#r26418417>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr